AllReadyApp/Web-App/AllReady/Areas/Admin/Controllers/CampaignController.cs

```diff
@@ -117,11 +119,16 @@ public IActionResult Create()
                         ModelState.AddModelError("ImageUrl", "You must upload a valid image file for the logo (.jpg, .png, .gif)");
                         return View(campaign);
                     }
+
+                    if (existingImageUrl != null)
+                    {
+                        await _imageService.DeleteImageAsync(existingImageUrl);
+                    }
                 }

```

Comment:
Steve, how does this look?
```
var existingImageUrl = campaign.ImageUrl;
if (fileUpload.IsAcceptableImageContentType())
{
     campaign.ImageUrl = await _imageService.UploadCampaignImageAsync(campaign.OrganizationId, campaign.Id, fileUpload);
    if (campaign.ImageUrl != null && existingImageUrl != null)
    {
        await _imageService.DeleteImageAsync(existingImageUrl);
    }
}
```

we're checking two things before invoking delete:
1. if there is an existing ImageUrl for the Campaign/Event, we have to capture that in order to know which image to delete after we've uploaded the new image.
2. the check you recommended in the code review, that the campaign.ImageUrl is not null, meaning that UploadCampaignImageAsync was successful.